4-projects/business_unit_1/shared/example_app_pipeline.tf

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "app_infra_cloudbuild_project" {
+  source                      = "../../modules/single_project"
+  impersonate_service_account = var.terraform_service_account
+  org_id                      = var.org_id
+  billing_account             = var.billing_account
+  folder_id                   = data.google_active_folder.common.name
+  environment                 = "shared"
+  alert_spent_percents        = var.alert_spent_percents
+  alert_pubsub_topic          = var.alert_pubsub_topic
+  budget_amount               = var.budget_amount
+  project_prefix              = var.project_prefix
+  activate_apis               = ["cloudbuild.googleapis.com", "sourcerepo.googleapis.com", "cloudkms.googleapis.com","binaryauthorization.googleapis.com","artifactregistry.googleapis.com","containeranalysis.googleapis.com","storage.googleapis.com"]
+
+  # Metadata
+  project_suffix    = "sample-infra"
+  application_name  = "app-infra-pipelines"
+  billing_code      = "1234"
+  primary_contact   = "example@example.com"
+  secondary_contact = "example2@example.com"
+  business_code     = "bu1"
+}
+
+module "infra_pipelines" {
```

Comment:
you may need to create an `app_pipelines` submodule as the infra_pipeline makes 2 triggers per csr, Terraform image etc and the triggers are configured with expectation that it will use the Terraform runner image